src/ContextTypes/NutritionInformation.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace JsonLd\ContextTypes;
+
+class NutritionInformation extends AbstractContext
```

Comment:
seems a child of'Thing' ?
https://schema.org/NutritionInformation

and test? :p